dist/_include-media.scss

```diff
@@ -94,6 +94,89 @@ $unit-intervals: (
   'rem': 0.1
 ) !default;
 
+///
+/// Defines whether support for media queries is available, useful for creating separate stylesheets
+/// for browsers that don't support media queries
+///
+/// @example scss - Disables support for media queries
+///  $im-media-support: false;
+///   @include media(">=tablet") {
+///     .foo {
+///       color: tomato;
+///     }
+///   }
+///
+///   /* Generates: */
+///   .foo {
+///     color: tomato;
+///   }
+///
+$im-media-support: true !default;
+
+///
+/// Selects which breakpoint to emulate when support for media queries is disabled. Media queries that start at or
+/// intercept the breakpoint will be displayed, any others will be ignored.
+///
+/// @example scss - This media query will show because it intercepts the static breakpoint
+///  $im-media-support: false;
+///   $im-no-media-breakpoint: 'desktop';
+///   @include media(">=tablet") {
+///     .foo {
+///       color: tomato;
+///     }
+///   }
+///
+///   /* Generates: */
+///   .foo {
+///     color: tomato;
+///   }
+///
+/// @example scss - This media query will NOT show because it does not intercept the desktop breakpoint
+///  $im-media-support: false;
```

Comment:
All the `@example` blocks throughout the code currently have the subsequent lines indented by one space. I think this was necessary to make SassDoc display the code blocks correctly, at least in the first version I've used.